.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "stage": 0
+  "presets": ["stage-0"]
```

Comment:
You don't need to use babel-cli or npm publish (this one is for publisher).
You don't need to compile to run test.
Just do "npm install" (you probably already did), and then "npm test". This will run test (lint + automated tests).
To see what happen under the hood, just type "npm run" and you should understand a bit what is happening.
Feel free to ask for more if you need.